ceph.spec.in

```diff
@@ -12,6 +12,10 @@
 %bcond_with selinux
 %endif
 
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12
+%if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version} == 1315
```

Comment:
Well, that comment said "RHEL 6+", and I thought that was the reason; if CentOS is now good, yeah, sure, but the comment should probably say so